.eslintrc.json

```diff
@@ -35,6 +35,7 @@
     "standard/no-callback-literal": 0,
     "comma-dangle": ["error", "always-multiline"],
     "space-before-function-paren": ["error", "never"],
+    "jsx-quotes": ["error", "prefer-double"],
```

Comment:
@skhameneh Do you know why this wouldn't be used? It's still using single quotes.

Reply:
Nevermind. Not sure what I was doing earlier but if works.